python/lsst/obs/decam/ingestCalibs.py

```diff
@@ -105,3 +82,26 @@ def getExtensionName(md):
         @param md (PropertySet) FITS header metadata
         """
         return md.get('EXTNAME')
+
+    def getDestination(self, butler, info, filename):
```

Comment:
The `getDestination` here is sufficiently different from the one in `pipe_tasks` that I think it makes sense to keep this here explicitly, but please let me know if I'm missing something.

I chatted with Merlin; he did try doing this, but it sounds like his problems occurred before ingestCalibs and it's not (easily) possible to test ingesting LSST-generated calibration data for decam because those calibration products don't exist yet. It's something I'd definitely like to revisit later!